bin/helm-deploy

```diff
@@ -41,6 +37,21 @@ if [ "${HELM_TEMPLATE_ONLY}" != "true" ]; then
   helm init --client-only
 fi
 
+detect_tiller_namespace() {
+    # If there is more than one tiller, then auto-detection will not work.
+    mapfile -t TILLERS < <(kubectl get deploy --all-namespaces -l app=helm,name=tiller | awk 'NR > 1 {print $1}')
+    if [ ${#TILLERS[@]} -gt 1 ]; then
+      echo "Multiple TILLERS were found.  This is not supported by auto-detection.  Please specify TILLER_NAMESPACE in your config."
+      exit 1
+    else
+      TILLER_NAMESPACE="${TILLERS[0]}"
+
+      # just in case variable didn't get set, give it a default
+      TILLER_NAMESPACE=${TILLER_NAMESPACE:-kube-system}
```

Comment:
Not a show stopper but is it possible to set the default in line 47?

Reply:
Totally.  Good call.